apps/src/lib/util/firehose.js

```diff
@@ -0,0 +1,114 @@
+/** @file Provides clients to AWS Firehose, whose data is imported into AWS Redshift. */
+
+import AWS from 'aws-sdk';
+
+const IDENTITY_POOL_ID = 'us-east-1:fbfec393-0afb-4682-84d4-59ad04a302f4'
+const AWS_REGION = 'us-east-1'
+
+AWS.config.region = AWS_REGION;
```

Comment:
We may want to lazily initialize the AWS stuff, so that for page loads that don't actually make any calls out, we don't make the connection at all.